### ISSUE TITLE
Module issues

### ISSUE THREAD

**Issue by GEMI** [FormidableLabs/webpack-alternate-require-loader#1]
Hi,
This project fails on Windows. Steps to reproduce:

- `npm install`
- `npm run build-demo-wp`

Produces result:
```

> webpack-alternate-require-loader@0.0.2 build-demo-wp C:\Users\GM\Documents\webpack-alternate-require-loader
> npm run clean && cd demo && webpack


> webpack-alternate-require-loader@0.0.2 clean C:\Users\GM\Documents\webpack-alternate-require-loader
> rimraf demo/build*

Hash: 13d15d66c8c7a9bf4b7b
Version: webpack 1.14.0
Time: 94ms
  Asset     Size  Chunks             Chunk Names
main.js  2.36 kB       0  [emitted]  main
   [0] ./main.js 611 bytes {0} [built] [2 errors]

ERROR in ./main.js
Module not found: Error: Cannot resolve module 'C:UsersGMDocumentswebpack-alternate-require-loaderdemo
ested
ode_modules
           ooindex.js' in C:\Users\GM\Documents\webpack-alternate-require-loader\demo
 @ ./main.js 13:10-129

ERROR in ./main.js
Module not found: Error: Cannot resolve module 'C:UsersGMDocumentswebpack-alternate-require-loaderdemar.js' in C:\Users\GM\Documents\webpack-alternate-require-loader\demo
 @ ./main.js 16:10-103
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! webpack-alternate-require-loader@0.0.2 build-demo-wp: `npm run clean && cd demo && webpack`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the webpack-alternate-require-loader@0.0.2 build-demo-wp script 'npm run clean && cd demo && webpack'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the webpack-alternate-require-loader package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run clean && cd demo && webpack
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs webpack-alternate-require-loader
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls webpack-alternate-require-loader
npm ERR! There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\GM\AppData\Roaming\npm-cache\_logs\2017-04-05T07_22_11_752Z-debug.log

```

Node version: 7.8.0 32bit
Npm version: 4.4.4

**Comment by GEMI** [comment]
The issue is that require fails because of the double backslashes.

```
   // Resolve path.
    var resolved = resolver.resolve(modPath);
    if (process.platform.indexOf("win") === 0) {
      resolved = resolved.replace(/\\/g, "/");
    }
```

**Comment by ryan-roemer** [comment]
Can you paste your webpack configuration snippet where this plugin is used? Aka something like that in: https://github.com/FormidableLabs/webpack-alternate-require-loader#examples

**Comment by ryan-roemer** [comment]
I suspect that your upstream webpack configuration may be the appropriate place for the transform you're trying here, aka something like:

```js
var os = require("os");

module.exports = {
  module: {
    loaders: [
      {
        test: /\.js$/,
        loader: "webpack-alternate-require-loader",
        query: JSON.stringify({
          "./outside-of-resolution-path/require": require
            .resolve("./outside-of-resolution-path/require")
            .replace(os.EOL, "/") // os-agnostic way to renormalize for require path.
        })
      }
    ]
  }
};
```

If that is the case, I'd definitely take a PR to update the README section of https://github.com/FormidableLabs/webpack-alternate-require-loader#examples with an additional example for the windows platform.

**Comment by ryan-roemer** [comment]
@GEMI -- Thanks for diagnosis and repro steps. I've opened #3 PR to hopefully fix this!

**Comment by GEMI** [comment]
@ryan-roemer no problem. Hope this helps. After fixing this issue and the builder issue I can finally launch my real project :)

**Comment by ryan-roemer** [comment]
Fixed in `webpack-alternate-require-loader@0.0.3`
